app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list) -> None:
+    [Elf.play_elf_song(elf) for elf in elves]
```

Comment:
first: You don't use a list which is created by list comprehension here, so u just waste memory. Comprehension is good to create lists, but `for loop` is better to call functions
second: Use instance of `Elf` to call method, not a class name

Fix everywhere